src/sentry/feedback/blueprints/api.md

```diff
@@ -0,0 +1,315 @@
+| device.family     | optional[string] | -                                                   |
+| device.model      | optional[string] | -                                                   |
+| device.name       | optional[string] | -                                                   |
+| dist              | optional[string] |                                                     |
+| environment       | optional[string] |                                                     |
+| feedback_id       | string           |                                                     |
+| locale            | object           | Browser locale information.                         |
+| locale.lang       | string           | The language preference of the user's browser.      |
+| locale.timezone   | string           | The timezone the feedback was submitted from.       |
+| message           | string           | The message written by the user providing feedback. |
+| os                | object           | -                                                   |
+| os.name           | optional[string] | -                                                   |
+| os.version        | optional[string] | -                                                   |
+| platform          | string           |                                                     |
+| project_id        | string           |                                                     |
+| release           | optional[string] |                                                     |
+| replay_id         | optional[string] | The id of a running replay.                         |
+| sdk               | object           | SDK information.                                    |
+| sdk.name          | string           | -                                                   |
+| sdk.version       | string           | -                                                   |
+| status            | string           | One of: resolved, unresolved.                       |
+| tags              | object           | Mapping of key, value pairs.                        |
+| timestamp         | string           | ISO-8061 formatted UTC datetime.                    |
+| url               | string           | The page the feedback was triggered on.             |
+| user              | object           | The authorized user's information.                  |
+| user.display_name | optional[string] | -                                                   |
+| user.email        | optional[string] | -                                                   |
+| user.id           | optional[string] | -                                                   |
+| user.ip           | optional[string] | Same search field as Events                         |
+| user.username     | optional[string] | -                                                   |
+
+- Response 200
+
+  ```json
+  {
+    "data": [
```

Comment:
can we add a `"count"` field to the response: return the number of records matching this query? 
I think that would be helpful for some potential interface ideas. I heard some of the Discover/Events endpoints have the same thing, and it should be a light-weight query, not something to do for replays for example.

Reply:
Of course.  Let's enumerate those requirements before we add it.

Reply:
relates to the figma design: https://www.figma.com/file/ga57zyLonVCSJIH3kOWLVA/Specs%3A-Bug-Reports?type=design&node-id=3-13295&mode=design&t=9FxkBVPALn6lqTuP-0

Counts at the top of the page: 
![SCR-20230921-knpi](https://github.com/getsentry/sentry/assets/187460/509e5e71-6966-49ad-bd9f-0297dec45054)


There are more ways to achieve that part of the design, so this is a suggestion